draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -196,25 +196,98 @@ Receipts MUST be tagged as COSE_Sign1.
 The following {{-CDDL}} definition is provided:
 
 ~~~ cddl
-Receipt = #6.18(COSE_Sign1)
+Signature_With_Receipt = #6.18(COSE_Sign1)
 
-cose-value = any
+cose.label = int / text
+cose.values = any
 
 Protected_Header = {
-  * cose-label => cose-value
+  * cose.label => cose.values
 }
 
 Unprotected_Header = {
-  &(receipts: 394)  => [+ bstr .cbor Receipt]
-  * cose-label => cose-value
+  &(receipts: 394)  => [+ bytes .cbor Receipt]
+  * cose.label => cose.values
 }
 
 COSE_Sign1 = [
-  protected   : bstr .cbor Protected_Header,
+  protected   : bytes .cbor Protected_Header,
   unprotected : Unprotected_Header,
-  payload     : bstr / nil,
-  signature   : bstr
+  payload     : bytes / nil,
+  signature   : bytes
 ]
+
+Receipt = Receipt_For_Inclusion / Receipt_For_Consistency
+
+; Note the the proof formats shown here are for RFC9162_SHA256.
+; Other verifiable data structures may have different proof formats.
+
+
+Receipt_For_Inclusion = #6.18(Signed_Inclusion_Proof)
+
+Signed_Inclusion_Proof = [
+  protected   : bytes .cbor Signed_Inclusion_Protected_Header
+  unprotected : Signed_Inclusion_Unprotected_Header
```

Comment:
I can see that this is done for consistency with `Signed_Inclusion_Proof` but it's unfortunate that something that is not signed has a name starting with `Signed`.

Reply:
I kinda agree, but am uncertain about a better approach. @achamayou, do you have a proposal? It is just an type name, but I see you point.